stylesheets/_component.dropzone.scss

```diff
@@ -0,0 +1,311 @@
+/**
+ * Dropzone
+ */
+
+$dropzone-file-size: .5em;
+$dropzone-thumb-size: 4em;
+
+.dropzone {
+    border: 2px dashed color(gray, light);
+    border-radius: $border-radius;
+    color: color(text, base);
+    padding: 1.25em;
+    position: relative;
+    text-align: center;
+}
+
+// the element responsible for creating success and error flashes
+.dropzone__state {
+    bottom: 0;
+    content: '';
+    height: 100%;
+    left: 0;
+    position: absolute;
+    pointer-events: none;
+    right: 0;
+    top: 0;
+    width: 100%;
+    z-index: 0;
+}
+
+.dropzone__file-wrapper {
+    display: flex;
```

Comment:
I should probably write some fallback code in, at the moment it falls back to everything being block, which looks acceptable, but should probably get some more love